### PR TITLE
Test/fixture cleanups (/more)

### PR DESCRIPTION
**Pull request by phlax**
Cleanup: remove old fixtures from changed_langs test

### CODE REVIEW

File: pytest_pootle/fixtures/models/store.py [hunk 1]
```diff
@@ -370,14 +370,6 @@ def ru_tutorial_po(settings, russian_tutorial):
 
 
 @pytest.fixture
-def ru_update_save_changed_units_po(settings, russian_tutorial):
-    """Require the /ru/tutorial/tutorial.po store."""
-    return _require_store(russian_tutorial,
-                          settings.POOTLE_TRANSLATION_DIRECTORY,
-                          'update_save_changed_units.po')
```
Comment (unho) [inline pytest_pootle/fixtures/models/store.py]: Just wondering if the files in https://github.com/phlax/pootle/tree/master/pytest_pootle/data/po/tutorial/ru should be dropped too.

Comment (phlax) [inline pytest_pootle/fixtures/models/store.py]: there are some other ru fixtures - so not sure tbh

Comment (unho) [inline pytest_pootle/fixtures/models/store.py]: If this specific file is not used anywhere else then I would say drop it.

Comment (phlax) [inline pytest_pootle/fixtures/models/store.py]: currently it is used - but if there are significant benefit to removing if that wasnt the case - the fiels in the tp folders are copied on every test

**Pull request merged**